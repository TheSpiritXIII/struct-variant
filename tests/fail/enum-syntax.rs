use struct_variant::struct_variant;

#[struct_variant]
enum ShapeEnumNoBound {
	Circle { x: i32, y: i32 },
	Rectangle(Rectangle, Rectangle),
}

#[struct_variant(Shape)]
enum ShapeEnumShapeBound {
	Circle { x: i32, y: i32 },
	Rectangle(Rectangle, Rectangle),
}

fn main() {}
