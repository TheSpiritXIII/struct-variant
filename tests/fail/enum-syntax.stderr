error: Failed to parse struct variant input: expected `,`
 --> $DIR/enum-syntax.rs:5:9
  |
5 |     Circle { x: i32, y: i32 },
  |            ^^^^^^^^^^^^^^^^^^

error: Failed to parse struct variant input: expected `,`
  --> $DIR/enum-syntax.rs:11:9
   |
11 |     Circle { x: i32, y: i32 },
   |            ^^^^^^^^^^^^^^^^^^
